index.html

```diff
@@ -328,12 +349,10 @@ <h1 class="contat">Contact me</h1>
 
```

Comment:
- [ ] [REQUIRED] Please remove `hr` tag on line `346` to `347` because it is not available in design 😃

Reply:
[Screencast from 07-10-2022 07:50:41 ALASIRI.webm](https://user-images.githubusercontent.com/81711768/194611122-988f0706-571d-4911-9e1a-1985097855bb.webm)

Hi reviewer, I think it would be better if you clone my project locally and then test it in your browser as the previous reviewer find some issues that on my end are working fine.
Thank

Reply:
@AbdimulhinYussuf3675 I cloned your project as you wished. You can rely the comments by the previous reviewer.